### DISCUSSION TITLE
Move the footer's social media buttons to the header with custom CSS?

### DISCUSSION THREAD

**Discussion by plasmidone** [GetPublii/Publii#1763] [Help]
Is it possible to move the social media buttons in the footer at the top instead of bottom of the page with custom CSS? I am using the Mercury theme 3.0.

**Comment by michaelfluegge** [comment]
I am not a programmer, but I think it is more than just CSS. 
Publii uses handlebars (*.hbs), so you may find a _footer.hbs_ in a folder _/partials_ in your theme folder. In the footer.hbs there should be a part like
```
{{#if @config.custom.socialButtons}}
```
and so on, which you maybe can move to the _head.hbs_ in the /partials folder. But I am very unsure whether it is that easy. It is kind of trial and error, I guess.

**Comment by bjazmoore** [comment]
@michaelfluegge is right.  You would need to modify the theme files.  Use theme overrides when changing the core theme files: https://getpublii.com/dev/theme-overrides/
